src/core/client/stream/tabs/Comments/Comment/MarkCommentAsSeenMutation.tsx

```diff
@@ -46,24 +43,28 @@ const enhanced = createMutation(
       variables: {
         input: {
           storyID: input.storyID,
-          commentID: input.commentID,
+          commentIDs: input.commentIDs,
           clientMutationId: clientMutationId.toString(),
         },
       },
       optimisticResponse: {
         markCommentAsSeen: {
-          comment: {
-            id: input.commentID,
-            seen: true,
-          },
+          comments: [
```

Comment:
Should the optimistic response include all comments provided in the input marked as seen?